INSTALL.md

```diff
@@ -10,6 +10,12 @@ This will also install any prerequisite Python packages.
 
 Since version 1.0.0, pywbem has no more OS-level prerequisite packages.
 
+On newer versions of some OS's (ex. Ubuntu 23.10, Debian 12) pywbem can only
+be installed into a virtual environment.  If an attempt is made to install it
+into the system directories the install will fail with a message about
```

Comment:
I suggest to use the term "into the system Python" instead of "into the system directories".

Also, I like the sentence from the readme file better than this sentence here: "This is by design to avoid conflicts ..."

Reply:
DONE